benchmarks/bvh_driver/benchmark_registration.hpp

```diff
@@ -0,0 +1,394 @@
+/****************************************************************************
+ * Copyright (c) 2017-2021 by the ArborX authors                            *
+ * All rights reserved.                                                     *
+ *                                                                          *
+ * This file is part of the ArborX library. ArborX is                       *
+ * distributed under a BSD 3-clause license. For the licensing terms see    *
+ * the LICENSE file in the top-level directory.                             *
+ *                                                                          *
+ * SPDX-License-Identifier: BSD-3-Clause                                    *
+ ****************************************************************************/
+
+#ifndef BENCHMARK_REGISTRATION_HPP
+#define BENCHMARK_REGISTRATION_HPP
+
+#include <ArborX_CrsGraphWrapper.hpp>
+#include <ArborX_Point.hpp>
+#include <ArborX_Predicates.hpp>
+
+#include <Kokkos_Core.hpp>
+
+#include <chrono>
+#include <cmath> // cbrt
+
+#include <benchmark/benchmark.h>
+#include <point_clouds.hpp>
+#include <spec.hpp>
+
+template <typename DeviceType>
+inline Kokkos::View<ArborX::Point *, DeviceType>
```

Comment:
```suggestion
Kokkos::View<ArborX::Point *, DeviceType>
```
and same below for `makeSpatialQueries` and `makeNearestQueries `

Reply:
Don't think so. We do multiple tests instantiations for the same backend. For example, for Serial we would instantiate `makeSpatialQueries` and `makeNearestQueries` for boost `BVH<Serial>` and `BoostRTree`, resulting in linking failure.

Reply:
The point is that templates practically behave the same as `inline` functions even without the additional attribute. For me, this comes down to a matter of style and being consistent (where I don't lean too much in one direction or the other).

Reply:
There would be no link failure because these are template.  The `inline` specifier does not hurt but I objected to your adding it.  It is hard to tell what was simply moved from one file to another and what changed.  No need to inject superfluous changes in the mix.

Reply:
While I don't understand how this works yet, I defer to you, and plan on reading through [stackoverflow](https://stackoverflow.com/questions/44335046/how-does-the-linker-handle-identical-template-instantiations-across-translation).